google/cloud/spanner_dbapi/parser.py

```diff
@@ -150,38 +149,29 @@ def parse_values(stmt):
 
 
 def expect(word, token):
-    word = word.strip(" ")
+    word = word.strip()
     if token == VALUES:
         if not word.startswith("VALUES"):
             raise ProgrammingError(
                 "VALUES: `%s` does not start with VALUES" % word
             )
-        word = word[len("VALUES") :].strip(" ")
+        word = word[len("VALUES") :].lstrip()
 
         all_args = []
         while word:
-            word = word.strip(" ")
+            word = word.strip()
 
             word, arg = expect(word, ARGS)
             all_args.append(arg)
+            word = word.strip()
 
-            word = word.strip(" ")
-            if word == "":
-                break
-            elif word[0] != ",":
+            if word and not word.startswith(","):
```

Comment:
As `while` circle already checks that `word` is not empty, we can reduce number of conditions here